packages/labs/analyzer/src/lib/analyze-package.ts

```diff
@@ -41,15 +41,16 @@ export const createPackageAnalyzer = (
   let commandLine: ts.ParsedCommandLine;
   if (ts.sys.fileExists(configFileName)) {
     const configFile = ts.readConfigFile(configFileName, ts.sys.readFile);
+    const existingOptions: CompilerOptions = {};
     if (options.exclude !== undefined) {
-      (configFile.config.exclude ??= []).push(...options.exclude);
+      existingOptions.exclude = options.exclude;
```

Comment:
Do you know for sure that this doesn't override the excludes in the config file? Maybe you should still concat the two?

Reply:
i don't know for sure, but internally, that param is called `existingOptions`, and it is processed in https://github.com/microsoft/TypeScript/blob/9701f55f7217d7bd56d28e73b250444efcefa8dd/src/compiler/commandLineParser.ts#L2859